core/templates/dev/head/css/oppia.css

```diff
@@ -3995,6 +4020,62 @@ md-card.preview-conversation-skin-supplemental-card {
   box-shadow: 0 6px 13px rgba(0,0,0,0.17), 0 4px 6px rgba(0,0,0,0.23);
 }
 
+.oppia-activity-summary-tile.oppia-activity-playlist-tile .title-section {
+  display: inline-block;
+  height: 7em;
+  margin: 16px 0 10px 16px;
+  min-width: 163px;
+  width: 27%;
+}
+
+.oppia-activity-summary-tile.oppia-activity-playlist-tile .summary-section {
+  display: inline-block;
+  float: right;
+  margin-top: 3%;
+  width: 67%;
+}
+
+.oppia-activity-summary-tile.oppia-activity-playlist-tile .activity-title {
+  font-size: 0.85em;
+  text-align: center;
+}
+
+@media(max-width: 895px) {
```

Comment:
Should these and some of the stuff below also have min-widths in order not to conflict with the rules that apply for smaller widths?

Reply:
I have tried and tested this. On reducing the width, the latest one always takes precedence.

Reply:
Yep, I know, but the existing CSS is fragile. Things get reordered. You might want to consider something like

`@media (min-width: 700px) and (max-width: 768px) {`

to make it more robust.

Reply:
Done!